testing/examples/test_examples.py

```diff
@@ -0,0 +1,71 @@
+import os
+import shutil
+
+from neuraxle.base import DEFAULT_CACHE_FOLDER
+
+
+def test_non_fittable_mixin():
+    os.makedirs(DEFAULT_CACHE_FOLDER)
+
+    from examples.non_fittable_mixin import main
+    main()
+
+    shutil.rmtree(DEFAULT_CACHE_FOLDER)
+
+def test_inverse_transform():
```

Comment:
See the test class `class TestConvolved1D(TestCase):
` here: 
- https://github.com/guillaume-chevalier/python-conv-lib/blob/master/testing/test_convolved_1d.py

This way you can define a [teardown](https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown) method on the class only once and it'll run for each test, even when they fail, to be sure to clean up.

Reply:
@guillaume-chevalier looks cool, but we are using pytest so that wouldnt work

Reply:
Yes that would: the test of `conv` I linked to you uses pytest as a test runner, but still imports unittest's TestCase class.